platform/backend/repository/testbed/src/test/kotlin/io/hamal/repository/CodeTest.kt

```diff
@@ -418,6 +417,41 @@ internal class CodeRepositoryTest : AbstractUnitTest() {
 
     }
 
+
+    @Nested
+    inner class CodeTypeTest {
```

Comment:
Code type sits in a complete different module --> move to package io.hamal.lib.domain.vo